src/class-rest.php

```diff
@@ -90,13 +90,19 @@ public function register_meta(): void {
 	 * response and the `meta` object containing the actual metadata.
 	 */
 	public function get_callback( array $object ): array {
-		$post_id = $object['ID'] ?? $object['id'];
+		$post_id = $object['ID'] ?? $object['id'] ?? '';
```

Comment:
`WP_Post::get_instance()` is expecting an integer, so passing an empty string doesn't seem right. The value gets cast to an integer anyway (so would be an integer `0`), so may as well just use `0` instead of an empty string here.

```suggestion
		$post_id = $object['ID'] ?? $object['id'] ?? 0;
```